### PR TITLE
improved main readme

### PR DESCRIPTION
**Pull request by worldofjoni**

### CODE REVIEW

**Comment by worldofjoni** [comment]
@Whatsuup @uglrl can you look throug your "warn" log messages and check whether they can be promoted to "error" according to the definition in the backend readme?

**Comment by worldofjoni** [comment]
Better images are still missing

**Comment by worldofjoni** [comment]
@uglrl maybe you can do some grammar and formulation checking 😁

**Review by AlexKutschera — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,25 +1,52 @@
...
+
+<a href='https://play.google.com/store/apps/details?id=de.mensa_ka.app&pli=1&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png' width=100em/></a>
+
 
 
 **Application for communication and collective exchange of
 menu information in university catering facilities.**
```
Comment (Whatsuup) [inline README.md:12]: I would still keep that "early state" as we'll change a lot after pse

**Review by ElenaHaeussler — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,25 +1,52 @@
...
 
-| 🚧        | This project is still in a early state and WIP!       |
-|------------|:-----------------------------------------------------------|
+The goal of this application is to provide easy access to the meal plans of the canteens of the [Studierendenwerk Karlsruhe](https://www.sw-ka.de/de/hochschulgastronomie/speiseplan) providing an Android-App (iOS is also planned) and a GraphQL API.
+
+<img src="doc/img/gallery_view.webp" alt="gallery view" width=30%/>
+<img src="doc/img/detail_view.webp" alt="meal detail view" width=30%/>
+<!-- TODO more and better images -->
```
Comment (worldofjoni) [inline README.md:17]: todo

**Review by uglrl — APPROVED** [review]

**Pull request merged**